### DISCUSSION TITLE
Memory is not released after indexing

### DISCUSSION THREAD

**Discussion by MyWebmasterD** [photoprism/photoprism#4543] [Q&A]
![Screenshot 2024-09-20 115621](https://github.com/user-attachments/assets/2740bfa1-a523-4934-a051-40e09cc70715)
Hello,
I'm using photoprism in a container in my qnap ts-473A.

After indexing images the RAM usage (6GB) is not released and continue to show as used by the app.

I think that only restarting the container will release it, but for sure there is something going on that is not working well.

I think the build is [240711-2197af848], community edition

**Comment by alexisLefebvre** [comment]
See:

- https://github.com/photoprism/photoprism/issues/633
